### DISCUSSION TITLE
API version 1.11.0 upgrade

### DISCUSSION THREAD

**Discussion by jah1007** [weather-gov/api#618] [General]
Just want to inform the community that the next version of API will be pushed to production early next week. This release will add ATOM specific URLs and will fix the off by 1,1 problem listed on our known issues page over at weather.gov/documentation. A notification will go out on weather.gov/notification when it is live.

**Comment by JJKraw** [comment]
Looks like this is live, at least for BOX locations.

**Reply by jah1007** [reply]
It is live but we ran into an issue that requires a a hotfix for the ATOM formatted feed. Once that update is done on Monday we will send out the notice for the release.
